integrations/standalone/src/mock.tsx

```diff
@@ -23,7 +23,7 @@ export async function start(): Promise<void> {
     <React.StrictMode>
       <ThemeContextProvider theme={theme}>
         <ClientContextInstance.Provider value={{ client: inscriptionClient }}>
-          <App pid={'1'} />
+          <App app='' pmv='' pid={'1'} />
```

Comment:
Can we provide these values somehow so that mock works as similar as standalone?
What does {'1'} mean here?

Reply:
don't know what should be the benefit of this...
The mock does nothing with this information, it is only here to quickly test and develop some ui stuff.

Reply:
My idea was that if we provide here some dummy information we can check if they are sent correctly to the mock server. 
Something like app='testApp' pmv="test$12'

Reply:
Hmm... yes this is maybe an idea. But I think this has nothing to do with the mock server. Because we don't need to test the mock server. Or better we cannot test such stuff in the mock server, because this info is never displayed anywhere on the ui. And the mock server is only in charge in the mock webtests...

But a general App.tsx or a custom hock unit test would be good for this. Also to test the error view (e.g. if an unknown type is delivered).

But I will check that in a later PR.